Data/Sys/GameSettings/NAC.ini

```diff
@@ -5,8 +5,8 @@
 
 [EmuState]
 # The Emulation State. 1 is worst, 5 is best, 0 is not set.
-EmulationStateId = 4
-EmulationIssues = Needs EFB to RAM for the pause menu (background and Link).
+EmulationIssues =
+EmulationStateId = 3
```

Comment:
This was changed to follow the settings on the Gamecube version, since apparently they share similar graphical requirements, this was partially why I had two separate commits, because I wasn't sure what the exact effect of that 2nd change was.